relay/blockchain/currency_network_events.py

```diff
@@ -55,6 +55,10 @@ def interest_rate_given(self):
     def interest_rate_received(self):
         return self._web3_event.get("args").get("_interestRateReceived", 0)
 
+    @property
+    def is_frozen(self):
+        return self._web3_event.get("args").get("_isFrozen", False)
```

Comment:
Why would this value not be set in 'args'? I think you already require the new version, so my guess is '_isFrozen` will be there. If that's the case, please remove the 'False'.